shuffle/cli/src/test.rs

```diff
@@ -127,3 +157,43 @@ fn run_deno_test(project_path: &Path, config: &NodeConfig) -> Result<()> {
         .wait()?;
     Ok(())
 }
+
+fn run_move_unit_tests(project_path: &Path) -> Result<()> {
+    let main_path = project_path.join("main");
+    Command::new("move")
+        .args(&["package", "test"])
+        .current_dir(main_path)
+        .spawn()
+        .expect(
+            "move unit testing failed to start, is it installed?\
+         cargo install --path language/tools/move-unit-test in diem repo.",
+        )
+        .wait()?;
+    Ok(())
+}
+
+#[derive(Debug, StructOpt)]
+pub enum TestCommand {
+    #[structopt(about = "Runs end to end test in shuffle")]
+    E2e {
+        #[structopt(short, long)]
+        project_path: Option<PathBuf>,
+    },
+
+    #[structopt(about = "Runs move move unit tests in project folder")]
+    Unit {
+        #[structopt(short, long)]
+        project_path: Option<PathBuf>,
+    },
+}
+
+pub fn handle_test_subcommands(cmd: TestCommand) -> Result<()> {
+    match cmd {
+        TestCommand::E2e { project_path } => {
+            handle(shared::normalized_project_path(project_path)?.as_path())
```

Comment:
1. Let's rename this method from `handle` to `run_e2e_tests` or whatever you deem appropriate.
2. Let's name the calling method `handle_test_subcommands` to `handle`